src/devices/Is31fl3731/samples/MatrixRgb.sample.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Please put all sample projects into separate folder (samples\sample1, samples\sample2, etc.). Having multiple .csproj files in the same folder causes problems on macos due to a compiler bug.

Reply:
I believe that there are multiple bindings that have that problem. Is this our new policy?